src/languages/en-US/commands/tools.json

```diff
@@ -300,6 +300,8 @@
 		"examples": ["Should I implement the #anime channel?"],
 		"explainedUsage": [["Question", "The question to vote on."]]
 	},
+	"voteContentNeeded": "{{REDCROSS}} You must give a topic for the poll.",
+	"voteReactionBlocked": "{{REDCROSS}} I was not able to add the reactions necessary for this command to work!",
```

Comment:
```suggestion
	"voteReactionBlocked": "{{REDCROSS}} I was not able to add the reactions necessary for this command to work because you have me blocked! 😢",
```